services/ble/Gatt.proto

```diff
@@ -18,3 +18,112 @@ message DisServiceData
     bytes ieeeRegulatoryCertificationDataList = 8 [(bytes_size) =  32];
     bytes pnpId = 9 [(bytes_size) =  7];
 }
+
+service GattServerDisService
+{
+    option (service_id) = 132;
+
+    rpc SetDisServiceData(DisServiceData) returns(Nothing) { option (method_id) = 1; }
+}
+
+message MtuSize
+{
+    uint32 value = 1;
+}
+
+message Handle
+{
+    uint32 value = 1;
+}
+
+message CharacteristicData
+{
+    Handle handle = 1;
+    bytes data = 2 [(bytes_size) = 512];
+}
+
+message HandleRange
+{
+    Handle startHandle = 1;
+    Handle endHandle = 2;
+}
+
+message Service
+{
+    bytes uuid = 1 [(bytes_size) = 16];
+    HandleRange handles = 2;
+}
+
+message Characteristic
+{
+    bytes uuid = 1 [(bytes_size) = 16];
+    Handle handle = 2;
+    Handle valueHandle = 3;
+    uint32 properties = 4;
+}
+
+message Descriptr
```

Comment:
```suggestion
message Descriptor
```

Reply:
See the later commint. Descriptor keyword is not allowed.